math/wxMaxima/Portfile

```diff
@@ -1,59 +1,109 @@
-PortSystem      1.0
-PortGroup       active_variants 1.1
-PortGroup       wxWidgets 1.0
-
-name            wxMaxima
-version         15.04.0
-revision        0
-maintainers     nomaintainer
-license         gpl
-platforms       darwin
-categories      math aqua
-description     Graphical user interface for Maxima
-long_description \
-                wxMaxima is a cross platform GUI for the computer algebra \
-                system maxima based on wxWidgets.
-homepage        http://wxmaxima.sourceforge.net/
-master_sites    sourceforge:project/[string tolower ${name}]/${name}/${version}
-livecheck.regex (?:[string tolower ${name}]|${name})-(\[a-z0-9.\]+)${extract.suffix}
-checksums       rmd160  04295f10ccd074d7069d00b1a0fe41884817687c \
-                sha256  0005029798703470333309fda29c731e79c85de1c7fc09b16cb87045a03aa4ba
-
-# Upstream changed the distname's spelling some time after 13.x.
-distname        wxmaxima-${version}
-
-wxWidgets.use   wxWidgets-3.0
-
-patchfiles      patch-src_main.cpp.diff \
-                patch-src_Dirstructure.h.diff \
-                patch-src_wxMaxima.cpp.diff
-
-depends_lib     port:libiconv \
-                port:libsdl \
-                port:libxml2 \
-                port:${wxWidgets.port} \
-                port:zlib
-depends_run     port:maxima
-
-require_active_variants gnuplot wxwidgets
-
-post-patch {
-    reinplace "s|@@PREFIX@@|${prefix}|" ${worksrcpath}/src/main.cpp
-}
+PortSystem          1.0
 
-depends_skip_archcheck  maxima
-configure.args  --enable-printing --with-wxdir=${wxWidgets.wxdir}
-build.target-append wxMaxima.app
+PortGroup           active_variants 1.1
+PortGroup           wxWidgets 1.0
 
-destroot {
-    xinstall -m 755 -d ${destroot}${applications_dir}
-    file copy ${worksrcpath}/wxMaxima.app ${destroot}${applications_dir}
+name                wxMaxima
+version             18.02.0
+revision            0
```

Comment:
We do not use a `revision` line for revision 0, so you may remove this line.

Reply:
I see. I will remove the revision line.